File: 2160-minimum-sum-of-four-digit-number-after-splitting-digits/2160-minimum-sum-of-four-digit-number-after-splitting-digits.java
class Solution {
    public int minimumSum(int num) {
        ArrayList<Integer> list = new ArrayList<>();
        while(num>0){
            list.add(num%10);
            num/=10;
        }
        Collections.sort(list);
        int first = list.get(0)*10+list.get(2);
        int second = list.get(1)*10+list.get(3);
        return first+second;
    }
}

File: 2160-minimum-sum-of-four-digit-number-after-splitting-digits/2160-minimum-sum-of-four-digit-number-after-splitting-digits.js
/**
 * @param {number} num
 * @return {number}
 */
var minimumSum = function(num) {
    let arr = [];
    while(num!=0){
        let rem = num%10;
        arr.push(rem);
        num = Math.floor(num/10);
    }
    arr.sort();
    return (arr[2]+arr[0]*10)+(arr[3]+arr[1]*10);
};